docs/device-data/data-types/pump-settings.md

```diff
@@ -5,23 +5,24 @@
 ## Table of Contents
 
 1. [Quick Summary](#quick-summary)
-2. [Type (`type`)](#type-type)
-3. [Active Schedule (`activeSchedule`)](#active-schedule-activeschedule)
-4. [Basal Schedules (`basalSchedules`)](#basal-schedules-basalschedules)
-5. [Blood Glucose Target (`bgTarget`)](#blood-glucose-target-bgtarget)
-6. [Carb Ratio (`carbRatio`)](#carb-ratio-carbratio)
-7. [Firmware Version (`firmwareVersion`)](#firmware-version-firmwareversion)
-8. [Hardware Version (`hardwareVersion`)](#hardware-version-hardwareversion)
-9. [Insulin Sensitivity (`insulinSensitivity`)](#insulin-sensitivity-insulinsensitivity)
-10. [Manufacturers (`manufacturers`)](#manufacturers-manufacturers)
-11. [Model (`model`)](#model-model)
-12. [Name (`name`)](#name-name)
-13. [Override Presets (`overridePresets`)](#override-presets-overridepresets)
-14. [Serial Number (`serialNumber`)](#serial-number-serialnumber)
-15. [Software Version (`softwareVersion`)](#software-version-softwareversion)
-16. [Units (`units`)](#units-units)
-17. [Examples](#examples)
-18. [Keep Reading](#keep-reading)
+1. [Type (`type`)](#type-type)
```

Comment:
I use VS Code with [Markdown All In One](https://github.com/yzhang-gh/vscode-markdown) which auto-generates TOCs :)

Reply:
It annoyed me that this ordered list would have to be updated every time you wanted to add a new item anywhere, so I changed it to all `1.`s. 

And then I just learned that Markdown doesn't care at all what numbers you use, as long as the first one is 1: https://www.markdownguide.org/basic-syntax/#ordered-lists